### DISCUSSION TITLE
calibre in cask repo shows 7.1 but on the web is listed 6.29.0

### DISCUSSION THREAD

**Discussion by luzpaz** [Homebrew/discussions#4982] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

https://formulae.brew.sh/cask/calibre shows claibre as 
Current version: [6.29.0](https://download.calibre-ebook.com/6.29.0/calibre-6.29.0.dmg)
![Screenshot_20231215_053914](https://github.com/Homebrew/discussions/assets/4140247/9ab50621-c837-4db1-907f-71dc2ad3ec0c)

An yet the cask formula is at 7.1 https://github.com/Homebrew/homebrew-cask/commit/b92e7ae22048c6c033a442539dd5608b3b72f6ce
https://github.com/Homebrew/homebrew-cask/blob/master/Casks/c/calibre.rb
![Screenshot_20231215_054056](https://github.com/Homebrew/discussions/assets/4140247/11c86535-db44-41f4-b593-7909bb9e36e3)

What's going on ?

**Comment by cho-m** [comment]
See https://github.com/Homebrew/formulae.brew.sh/issues/965
And https://github.com/Homebrew/homebrew-cask/blob/master/Casks/c/calibre.rb#L34-L35
